src/app/lib/helpers.ts

```diff
@@ -50,3 +50,5 @@ export function concat(...parts: Uint8Array[]) {
 }
 
 export const parseNumberToBigInt = (value: number) => BigInt(Math.round(value * 10 ** 9))
+
+export var parseBigIntToString = (value: string) => (Number(value) / 10 ** 9).toString()
```

Comment:
could lose precision

Reply:
I moved this from current ReclaimEscrowForm `Number(props.maxAmount) / 10 ** 9`  and we do this in other places as well ('AmountFormatter') . Can you give some examples so I can dig into that more?

Reply:
ok if its moved from original code, I guess.

but fyi, for example `Number('10000000000000001')` ~10 mil ROSE goes wrong, due to limits of the number type

Reply:
Could we finally push forward this feature? Did you consider this workaround:
1. increase number of decimals to 10 for the amount in ROSE field
2. 100% button takes the number of shares as the source information and then computes the amount in ROSE